### PR TITLE
Add top_targets to explain_prediction

### LINKED ISSUE #149: Limit show_prediction only to few classes
I have 300 target classes and showing explanation of all of them make my browser very slow.
it could be very good if show_prediction could show the explanation of the 3 best predictions and not all the classes (y).
How can I do that?

### PR DESCRIPTION
**Pull request by lopuhin**
Fixes #149.
I think I added it to all explain_prediction functions, and also updated the notebooks that had a hack to show the top target.

### CODE REVIEW

File: eli5/explain.py [hunk 1]
```diff
@@ -106,6 +106,14 @@ def explain_prediction(estimator, doc, **kwargs):
 
         This argument may be supported or not, depending on estimator type.
 
+    top_targets : int, optional
+        Number of targets to show. When ``top_targets`` is provided,
+        only specified number of targets with highest scores are shown.
```
Comment (kmike) [inline eli5/explain.py]: What do you think about supporting negative top_targets? E.g. -1 would mean 1 target with lowest score. I'm not sure it is important for classification, but it could be useful for regression.

Comment (lopuhin) [inline eli5/explain.py]: 👍  I think it's a great idea, and negative values seem had to confuse with anything else.

File: eli5/explain.py [hunk 2]
```diff
@@ -106,6 +106,14 @@ def explain_prediction(estimator, doc, **kwargs):
 
         This argument may be supported or not, depending on estimator type.
 
+    top_targets : int, optional
```
Comment (kmike) [inline eli5/explain.py]: It should also be documented in show_prediction

File: eli5/explain.py (hunk 1 above)
Comment (lopuhin) [inline eli5/explain.py]: Done in 8b8010f

File: eli5/explain.py (hunk 2 above)
Comment (lopuhin) [inline eli5/explain.py]: Eh, I always forget about it! Fixed in 8b8010f

Comment (kmike) [inline eli5/explain.py]: It is not your fault, there is a lot of duplication in eli5 docs

**Comment by kmike** [comment]
Looks great, thanks!
